src/Pusher.php

```diff
@@ -406,6 +431,8 @@ public static function array_implode($glue, $separator, $array)
      * @param bool         $debug           [optional]
      * @param bool         $already_encoded [optional]
      *
+     * @throws PusherException throws exception if $channels is an array of size 101 or above or $socket_id is invalid
```

Comment:
nitpick: some descriptions you have upper cased the first letter, and some you have not. Could upper-case all of them please?

Reply:
yes, of cause, sorry